content/integrations/github.md

```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
   * Identify when new code changes lead to system alerts or build failures
   * Discuss code changes with your team in the Datadog Event Stream
 
-
+{{< img src="git_integration_screen.png" alt="Git event" >}}
 
 ## Configuration
 
+1. Add this URL to your Github Webhook, **Settings>Webhooks>Add new Webhook** with this URL:
```

Comment:
"In Github, add this URL to your webhooks (**Settings>Webhooks>Add new Webhook**):"